src/parser/smt2/smt2_state.cpp

```diff
@@ -159,6 +159,15 @@ void Smt2State::addDatatypesOperators()
     defineVar("tuple.unit", d_solver->mkTuple({}));
     addIndexedOperator(Kind::UNDEFINED_KIND, "tuple.select");
     addIndexedOperator(Kind::UNDEFINED_KIND, "tuple.update");
+    Sort btype = d_solver->getBooleanSort();
+    defineVar("nullable.null",
+              d_solver->mkNullableNull(d_solver->mkNullableSort(btype)));
+    addOperator(Kind::APPLY_CONSTRUCTOR, "nullable.some");
+    addOperator(Kind::APPLY_CONSTRUCTOR, "nullable.val");
+    addOperator(Kind::NULLABLE_LIFT, "nullable.lift");
+    addOperator(Kind::APPLY_CONSTRUCTOR, "nullable.is_null");
```

Comment:
Can you make this `APPLY_TESTER`? Ditto for `is_some`.

Reply:
Done